pkg/kor/kor.go

```diff
@@ -27,6 +28,11 @@ type IncludeExcludeLists struct {
 	ExcludeListStr string
 }
 
+type Config struct {
```

Comment:
Interesting. Did you have something in mind to support using this struct?

Reply:
It is used by `unmarshalConfig`. But happy to refactor that into not using it. The idea is that this sets us up in a way we could break out exceptions into individual cloud if we want in the future. Then we could adjust the code to do say GKE only (if you pass in a flag) or all by default.

Reply:
If we don't use a struct like this, then we will have to have each embedded file be its own struct in each of their individual files to unmarshal the data.

```
type Config struct {
	ExceptionStorageClasses []ExceptionResource `json:"exceptionStorageClasses"`
}

type ExceptionClusterRoles struct {
	ExceptionClusterRoles []ExceptionResource `json:"exceptionClusterRoles"`
}
```

Which will then require us to refactor the `unmarshalConfig` helper.

The other idea would be do away with JSON files altogether and just  create another Go package called `exceptions` and we move everything, i.e. `pkg/exceptions/exceptionStorageClasses.go`, `pkg/exceptions/exceptionClusterRoles.go`

Reply:
> It is used by `unmarshalConfig`. But happy to refactor that into not using it. The idea is that this sets us up in a way we could break out exceptions into individual cloud if we want in the future. Then we could adjust the code to do say GKE only (if you pass in a flag) or all by default.

I actually did not think of that. Really solid idea!

Reply:
I pushed up some more code to show how the `Config` struct will look in the future.

Reply:
Looks good to me. Go for it